tests/unit/models/test_templates.py

```
@@ -33,7 +32,7 @@ def test_api_data_should_return_dict_with_all_props(self) -> None:
 
 class TestUpdateEmailTemplateParams:
     def test_raise_error_when_all_fields_are_missing(self) -> None:
-        with pytest.raises(ValidationError) as exc:
+        with pytest.raises(ValueError) as exc:
             _ = UpdateEmailTemplateParams()
 
         assert "At least one field must be provided for update actio" in str(exc)
```

Ruff (0.12.2)

Use of `assert` detected

(S101)